### PR TITLE
1.19.1-pre3 chat mappings

### PR DESCRIPTION
**Pull request by mschae23**
Mappings for some of the chat changes and the new packet in 1.19.1-pre3.

### CODE REVIEW

**Review by apple502j — APPROVED** [review]
LGTM. Note that I also mapped them and will intend to rename some of the names here, namely the plurality of the new packet's name.

**Review by enbrain — APPROVED** [review]

File: mappings/net/minecraft/text/Decoration.mapping [hunk 1]
```diff
@@ -30,6 +30,16 @@ CLASS net/minecraft/class_7463 net/minecraft/text/Decoration
 		COMMENT @implNote This decoration allows use of the team name, the sender, and the
 		COMMENT content parameters. It has no style.
 		ARG 0 translationKey
+	METHOD method_44779 ofIncomingMessage (Ljava/lang/String;)Lnet/minecraft/class_7463;
+		COMMENT {@return the decoration used in chat messages}
+		COMMENT
+		COMMENT @implNote This decoration allows use of the sender and the content parameters. It is gray and italic.
```
Comment (Juuxel) [inline mappings/net/minecraft/text/Decoration.mapping]:
```suggestion
		COMMENT @implNote This decoration allows using the sender and the content parameters. It is gray and italic.
```
I think "use of" without an article is ungrammatical? "Using" is more concise anyway

File: mappings/net/minecraft/text/Decoration.mapping [hunk 2]
```diff
@@ -30,6 +30,16 @@ CLASS net/minecraft/class_7463 net/minecraft/text/Decoration
...
+		ARG 0 translationKey
+	METHOD method_44780 ofOutgoingMessage (Ljava/lang/String;)Lnet/minecraft/class_7463;
+		COMMENT {@return the decoration used in chat messages}
+		COMMENT
+		COMMENT @implNote This decoration allows use of the target and the content parameters. It is gray and italic.
```
Comment (Juuxel) [inline mappings/net/minecraft/text/Decoration.mapping]:
```suggestion
		COMMENT @implNote This decoration allows using the target and the content parameters. It is gray and italic.
```

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/text/Decoration.mapping (hunk 1 above)
Comment (mschae23) [reply mappings/net/minecraft/text/Decoration.mapping]: I just copied the documentation of the other message types for this. Should they be changed as well?

Comment (apple502j) [reply mappings/net/minecraft/text/Decoration.mapping]: I can fix it on my PR if you want, or you can fix it here :)

Comment (mschae23) [reply mappings/net/minecraft/text/Decoration.mapping]: Okay, I fixed it.

**Review by Juuxel — APPROVED** [review]

**Pull request merged**